OneCommunityGlobal/HighestGoodNetworkApp
# Jiayu --> Miguel Adding Issue Form

# Description

Design is based on the Figma prototype called **Issue Form** in Phase II. This is only a frontend change, so no backend functionality.

![figma_issue_log](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/14044314/6f615c1e-8ca5-469f-9ab7-77280ba0bef8)

For Issue Type dropdown options the following should appear

-  Issue Type Dropdown options:
    -  Safety
    -  METs quality / functionality
    -  Labor
    -  Weather
    -  Other

Based on Issue Type, the Consequenc(s) options change depend on that:

- Safety
    - MET Damage/Waste
    - Schedule Delay
    - Minor Injury
    - Major Injury
    - Other

- METs Quality/Functionality
    - Require Reorder of MET
    - Schedule Delay
    - Other

- Labor
    - Quality Issue
    - Schedule Delay
    - Other

- Weather
    - Require Reorder of MET
    - Schedule Delay
    - Other

- Other
    - Other 


## Related PRS (if any):
N/A

## Main changes explained:
- Created route
- Created Issue Form based on Figma

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as any user
5. Visit [localhost:3000/bmdashboard/issue](http://localhost:3000/bmdashboard/issue) (Log in as same user if asked)
6. verify dropdown options, character limit.
## Screenshots or videos of changes:
![my_issue_form](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/14044314/41852887-0b92-4bbb-bc08-9ef8ba02c414)

## Note:
Basic frontend functionality. No actual saving works yet.
